public/icons/README.md

```diff
@@ -0,0 +1,9 @@
+# Icon Usage Guidelines
```

Comment:
:+1: for taking the time to write this. I suggest that we move it into the `/docs` folder to keep it a part of the general documentation.

If we need to then we can create a small README in this folder and then reference the larger file in `/docs`.

Reply:
In the future should expand the documentation to explain how developers should use the SVGs ie. inline the files when requiring animation using either copy/paste of techniques like `source()` in Twig and direct references with `<img/>` or `background-image` otherwise.